README.md

```diff
@@ -17,6 +17,7 @@ Creates a Raven Middleware.
  - `actionTransformer` -- transform the action before sending to Sentry
  - `stateTransformer` -- transform the state before sending to Sentry
  - `logger` -- log function to use instead of `console.error`
+ - `enableBreadcrumbsCapture` -- whether to send breadcrumbs as part of request
```

Comment:
Change this to disable. So it should be enabled by default as the author expected. Also, for backward compatibility. Others might be using the plugin. If your PR gets merged, then it will be a breaking change